### DISCUSSION TITLE
Is there a way to approve Weblate translations?

### DISCUSSION THREAD

**Discussion by MarioSwitch** [osmandapp/OsmAnd#20179] [General]
Now, only English and Ukrainian translations are approved on Weblate.
Is there a way to approve other completed languages translations (>90% translated) or is it planned?
And if not, would you like to add some very active contributors as "approvers"?
Thank you!

**Comment by ZeljkoBG** [comment]
OsmAnd adds the latest translations to the application. Whether approved or not.
